coordinator/src/app.ts

```diff
@@ -2,14 +2,15 @@ import {Boom} from "@hapi/boom"
 import Hapi from "@hapi/hapi"
 import routes from "./routes"
 import {toOperationOutcome, FhirMessageProcessingError} from "./models/errors/processing-errors"
+import HapiPino from "hapi-pino"
 
 const preResponse = function (request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit) {
   const response = request.response
   if (response instanceof Boom) {
```

Comment:
I think you need to swap these conditions round and handle `FhirMessageProcessingError` first

Reply:
Double checking now

Reply:
Yeah, looks like we're now returning 500s for validation errors where we used to return 400s

Reply:
Pushed a fix